website/docs/r/online_archive.html.markdown

```diff
@@ -81,28 +88,37 @@ resource "mongodbatlas_online_archive" "test" {
 * `paused`           - (Optional) State of the online archive. This is required for pausing an active or resume a paused online archive. The resume request will fail if the collection has another active online archive.
 
-### Criteria details
+### Criteria
 
 There are two types of criteria, `DATE` to select documents for archiving based on a date and
 `CUSTOM` to select documents for archiving based on a custom JSON query.
 
-* `criteria.type`          - Type of criteria (DATE, CUSTOM)
+* `type`          - Type of criteria (DATE, CUSTOM)
 
 The following fields are required for criteria type `DATE`
 
-* `criteria.date_field`    - Name of an already indexed date field from the documents. Data is archived when the current date is greater than the value of the date field specified here plus the number of days specified via the `expire_after_days` parameter.
-* `criteria.date_format`   - the date format. Valid values:  ISODATE (default), EPOCH_SECONDS, EPOCH_MILLIS, EPOCH_NANOSECONDS
-* `criteria.expire_after_days` - Number of days that specifies the age limit for the data in the live Atlas cluster. Data is archived when the current date is greater than the value of the date field specified via the `date_field` parameter plus the number of days specified here.
+* `date_field`   - Indexed database parameter that stores the date that determines when data moves to the online archive. MongoDB Cloud archives the data when the current date exceeds the date in this database parameter plus the number of days specified through the expireAfterDays parameter.
+* `date_format`   - Syntax used to write the date after which data moves to the online archive. Date can be expressed as ISO 8601 or Epoch timestamps. The Epoch timestamp can be expressed as nanoseconds, milliseconds, or seconds. You must set `type` to `DATE` if `collectionType` is `TIMESERIES`. Valid values:  ISODATE (default), EPOCH_SECONDS, EPOCH_MILLIS, EPOCH_NANOSECONDS.
+* `expire_after_days` - Number of days after the value in the criteria.dateField when MongoDB Cloud archives data in the specified cluster.
 
 The only field required for criteria type `CUSTOM`
 
-* `criteria.query` - JSON query to use to select documents for archiving. Atlas uses the specified query with the db.collection.find(query) command. The empty document {} to return all documents is not supported.
+* `query` - JSON query to use to select documents for archiving. Atlas uses the specified query with the db.collection.find(query) command. The empty document {} to return all documents is not supported.
+
+### Schedule
```

Comment:
`schedule` should have also been added to the `Argument Reference` section above.